android/build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     ext.assertj_version = '3.22.0'
     ext.mockk_version = '1.12.3'
     ext.gradle_maven_publish = '0.19.0'
-    ext.purchases_version = '5.4.1'
+    ext.purchases_version = '5.6.0'
```

Comment:
This should match the next release since we're adding `setCleverTapID` to `android` and we're already at `5.5.0`

Reply:
5.6.0 has been released